Unhandled Exception in ``get_all_metadata_index_responders()``

I noticed a few unhandled errors in the logs. These requests:

```
127.0.0.1 - - [25/Jul/2017:22:36:42 -0700] "GET /pcds/agg/?from-date=2017%2F01%2F01&to-date=2017%2F12%2F31&input-polygon=&input-var=&network-name=MoTIe&input-freq=1-hourly&data-format=ascii&cliptodate=cliptodate&download-timeseries=Timeseries HTTP/1.1" 200 9754965 "-" "WeatherUpdater"
127.0.0.1 - - [25/Jul/2017:22:36:52 -0700] "GET /pcds/agg/?from-date=2017%2F01%2F01&to-date=2017%2F12%2F31&input-polygon=&input-var=&network-name=BCH&input-freq=daily&data-format=ascii&cliptodate=cliptodate&download-timeseries=Timeseries HTTP/1.1" 200 107642 "-" "WeatherUpdater"
```

Seem to correspond to these errors:

```
[2017-07-25 22:36:43 +0000] [27692] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:43 +0000] [13732] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:43 +0000] [13742] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:43 +0000] [27692] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:52 +0000] [13756] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:53 +0000] [13742] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:53 +0000] [13742] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:53 +0000] [13748] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:53 +0000] [28948] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:53 +0000] [28948] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:53 +0000] [28948] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
[2017-07-25 22:36:53 +0000] [27692] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 97, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
```

We continue to see this crop up. E.g.:

```
2018-08-13 05:17:05 [15] [ERROR] Error handling request
Traceback (most recent call last):
  File
"/usr/local/lib/python2.7/dist-packages/gunicorn/workers/async.py", line
114, in handle_request
    for item in respiter:
  File "/root/pdp/pdp/error.py", line 74, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/http/wsgi.py",
line 237, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/root/pdp/pdp/error.py", line 68, in __call__
    for block in response_iter:
  File 
"/usr/local/lib/python2.7/dist-packages/ga_wsgi_client/__init__.py",
line 41, in __call__
    for chunk in app_iter:
  File "/usr/local/lib/python2.7/dist-packages/pdp_util/agg.py", line
76, in ziperator
    for name, responder in responders:
  File "/usr/local/lib/python2.7/dist-packages/pdp_util/agg.py", line
94, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
2018-08-13 05:17:05 [15] [ERROR] Error handling request
Traceback (most recent call last):
  File
"/usr/local/lib/python2.7/dist-packages/gunicorn/workers/async.py", line
114, in handle_request
    for item in respiter:
  File "/root/pdp/pdp/error.py", line 74, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/http/wsgi.py",
line 237, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/root/pdp/pdp/error.py", line 68, in __call__
    for block in response_iter:
  File 
"/usr/local/lib/python2.7/dist-packages/ga_wsgi_client/__init__.py",
line 41, in __call__
    for chunk in app_iter:
  File "/usr/local/lib/python2.7/dist-packages/pdp_util/agg.py", line
76, in ziperator
    for name, responder in responders:
  File "/usr/local/lib/python2.7/dist-packages/pdp_util/agg.py", line
94, in get_all_metadata_index_responders
    nets = set(zip(*stations)[0])
IndexError: list index out of range
```

--- a/.github/workflows/python-ci.yml
+++ b/.github/workflows/python-ci.yml
@@ -20,6 +20,7 @@ jobs:
       env:
         PIP_INDEX_URL: https://pypi.pacificclimate.org/simple
       run: |
+        sudo apt purge postgresql-client-13 postgresql-server-dev-13
         sudo apt-get install libhdf5-serial-dev libnetcdf-dev libspatialite-dev postgresql-9.5-postgis-2.4
         pip install -r requirements.txt -r test_requirements.txt
         pip install .

--- a/docker/local-pytest/README.md
+++ b/docker/local-pytest/README.md
@@ -110,7 +110,7 @@ you should not need to do this. However, just in case:
 From the _project root directory_ (important Docker context location):
 
 ```
-docker build -t pcic/pdp_util-local-pytest -f docker/local-test/Dockerfile .
+docker build -t pcic/pdp_util-local-pytest -f docker/local-pytest/Dockerfile .
 ```
 
 If you are having trouble installing packages that should be there, try using

--- a/pdp_util/agg.py
+++ b/pdp_util/agg.py
@@ -91,10 +91,10 @@ def get_all_metadata_index_responders(sesh, stations, climo=False):
     :type climo: bool
     :rtype: iterator
     '''
-    nets = set(zip(*stations)[0])
-    for net in nets:
-        filename = '{0}/variables.csv'.format(net)
-        yield (filename, metadata_index_responder(sesh, net, climo))
+    for station in stations:
+        network_name = station[0]
+        filename = '{0}/variables.csv'.format(network_name)
+        yield (filename, metadata_index_responder(sesh, network_name, climo))
 
 def metadata_index_responder(sesh, network, climo=False):
     '''The function creates a pydap csv response which lists variable metadata out of the database. It returns an generator for the contents of the file

--- a/tests/test_agg.py
+++ b/tests/test_agg.py
@@ -31,18 +31,36 @@ def test_metadata_index_responder(test_session):
     for line in lines:
         assert line in response_text
 
-# This seems dumb to test, but it's cheap
-def test_get_all_metadata_index_responders(test_session, monkeypatch):
+
+@pytest.mark.parametrize(
+    "stations, expected_filenames",
+    [
+        ([], set()),
+        (
+            stns,
+            set([
+                'ARDA/variables.csv',
+                'EC_raw/variables.csv',
+                'FLNRO-WMB/variables.csv'
+            ])
+        ),
+    ]
+)
+def test_get_all_metadata_index_responders(
+    test_session,
+    monkeypatch,
+    stations,
+    expected_filenames,
+):
     # Content is tested elsewhere. Fake it out.
     def fake_metadata_index_responder(sesh, net, climo):
         return []
     monkeypatch.setattr(pdp_util.agg, 'metadata_index_responder', fake_metadata_index_responder)
 
-    expected_filenames = set(['ARDA/variables.csv', 'EC_raw/variables.csv', 'FLNRO-WMB/variables.csv'])
-
-    resp = get_all_metadata_index_responders(test_session, stns, False)
+    resp = get_all_metadata_index_responders(test_session, stations, False)
     filenames = set([filename for filename, content in resp])
     assert filenames == expected_filenames
+
 
 def test_get_pcds_responders(conn_params, monkeypatch):
     # Don't care about content (tested elsewhere); just return the environ

